simsx/delivery.go

```diff
@@ -0,0 +1,98 @@
+package simsx
```

Comment:
i dont follow the x suffix, it feels odd

Reply:
Yes, it feels experimental